docs/src/examples/redis-clustering.md

```diff
@@ -2,72 +2,52 @@
-```
-
-If Shotover can successfully contact your Redis cluster, you should see the following:
-
+Run shotover, if Shotover can successfully contact your Redis cluster you should see the following:
 ```console
-user@demo$ ./shotover-proxy --config-file redis-shotover.yml 
-Aug 17 12:11:42.867  INFO shotover_proxy: Loading configuration
-Aug 17 12:11:42.867  INFO shotover_proxy: Starting loaded topology
-Aug 17 12:11:42.867  INFO shotover_proxy::config::topology: Loaded topics ["testtopic"]
-Aug 17 12:11:42.876  INFO shotover_proxy::config::topology: Loaded chains ["redis_chain"]
-Aug 17 12:11:42.876  INFO shotover_proxy::sources::redis_source: Starting Redis source on [127.0.0.1:6379]
-Aug 17 12:11:42.878  INFO shotover_proxy::config::topology: Loaded sources [["redis_prod"]] and linked to chains
-Aug 17 12:11:42.878  INFO shotover_proxy::server: accepting inbound connections
+user@demo$ ./shotover-proxy
+2021-11-03T06:20:03.955870Z  INFO shotover_proxy::runner: Starting Shotover 0.1.0
+2021-11-03T06:20:03.955879Z  INFO shotover_proxy::runner: configuration=Config { main_log_level: "info,shotover_proxy=info", observability_interface: "0.0.0.0:9001" }
+2021-11-03T06:20:03.955884Z  INFO shotover_proxy::runner: topology=Topology { sources: {"redis_prod": Redis(RedisConfig { listen_addr: "127.0.0.1:6379", connection_limit: None, hard_connection_limit: None, tls: None })}, chain_config: {"redis_chain": [RedisSinkCluster(RedisSinkClusterConfig { first_contact_points: ["127.0.0.1:2220", "127.0.0.1:2221", "127.0.0.1:2222", "127.0.0.1:2223", "127.0.0.1:2224", "127.0.0.1:2225"], tls: None, connection_count: None })]}, named_topics: {"testtopic": 5}, source_to_chain_mapping: {"redis_prod": "redis_chain"} }
+2021-11-03T06:20:03.955893Z  INFO shotover_proxy::config::topology: Loaded topics ["testtopic"]
+2021-11-03T06:20:03.956635Z  INFO shotover_proxy::transforms::redis::sink_cluster: connected to upstream
+2021-11-03T06:20:03.956644Z  INFO shotover_proxy::config::topology: Loaded chains ["redis_chain"]
+2021-11-03T06:20:03.956648Z  INFO shotover_proxy::sources::redis_source: Starting Redis source on [127.0.0.1:6379]
+2021-11-03T06:20:03.956659Z  INFO shotover_proxy::config::topology: Loaded sources [["redis_prod"]] and linked to chains
+2021-11-03T06:20:03.956675Z  INFO shotover_proxy::server: accepting inbound connections
 ```
 
-Currently the RedisSinkCluster transform, needs to be able to connect to the Redis cluster when it starts up. If it cannot, Shotover Proxy will exit with a panic, indicating it couldn't connect to the contact points. 
-
-Note: Currently `shotover-proxy` cannot daemonize itself. So you may wish to use a service supervisor to do this for you or you can simply run this in a different terminal session in development/testing scenarios :)
+Currently the RedisSinkCluster transform, needs to be able to connect to the Redis cluster when it starts up otherwise shotover will exit with an error.
```

Comment:
Code formatting on `RedisSinkCluster`

Reply:
Why do we say "Currently"? It implies it will/might change in the future

Reply:
I think we were actually were thinking of changing that but I agree, no need for that to be mentioned here.